packages/svelte/.eslintrc.cjs

```diff
@@ -21,6 +21,7 @@ module.exports = {
     rules: {
       'svelte/indent': ['error'],
       'no-use-before-define': 'off',
+      'no-undef-init': 'off',
```

Comment:
@rokotyan Context: The linter has been removing `= undefined` initializations in the pre-commit script. However for the sake of svelte components, it is necessary to include them because it signifies that a prop is optional (see https://svelte.dev/docs/svelte-components#script-1-export-creates-a-component-prop)
```
// Required prop
export let prop: PropType;

// Optional prop
export let prop: PropType = undefined
```

Previously the only consequence was misguided console warnings (for example _XYContainer is missing required prop `data`_) but now that I've added prop type defs for better IntelliSense, we need to include these initializations to prevent pipeline errors.

Reply:
@reb-dev Nice catch!